changelog.md

```diff
@@ -31,7 +19,7 @@ subfolder.
 * Fix typos in messages (#1651)
 * Fix homepage URL in setup.py (#1662)
 * Remove obsolete gvfs-bin dependency from securedrop-workstation-viewer (#1842)
-* Set umask 0o22 before writing to USBs (#1872)
+* Set umask 0o22 before writing to USBs so exported files are world readable (#1872, #1917)
```

Comment:
Exported files are now world-readable, but the umask portion is no longer applicable (sorry)  and we can remove it from the changelog

Reply:
Fixed!